### DISCUSSION TITLE
Panel Linking to change templating variables of the same dashboard

### DISCUSSION THREAD

**Discussion by ygutgutia** [grafana/grafana#40837] [Dashboards, Panels & Visualizations]
If I want to change the template-variables of the dashboard I am working on using panel links, sing the UI it's easy to do it. But if I intend to generate a similar dashboard using the HTTP API, my panel links will be broken because there is no way for me to know the UID of the new dashboard beforehand, and thus, I cant support panel links anymore.

I can create a UID myself, but I think there should be a provisioning to dynamically load the URL of the dashbaord I am working on(just like template-variables).

**Comment by torkelo** [comment]
Use template variable __dashboard.uid

**Reply by ygutgutia** [reply]
Oh thanks I didnt know this exists. This is awesome!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
